example/dynamic_sql_query_for_appointment.sql

```diff
@@ -52,7 +52,7 @@ SELECT a.id, CONCAT(b.site_short_name, ' - ', from_unixtime(a.appointment_block,
             AND (
             ( appointment_block > UNIX_TIMESTAMP(
                      -- If it is later than 3pm, only show appointments at least 2 days from today
-                    DATE( NOW() + INTERVAL IF(HOUR(NOW()) >= 15, 2, 1) DAY ) +
+                    DATE( NOW() + INTERVAL IF(HOUR(NOW()) >= 15 & WEEKDAY(NOW()) != 6, 2, 1) DAY ) +
```

Comment:
```suggestion
                    DATE( NOW() + INTERVAL IF(HOUR(NOW()) >= 15 AND WEEKDAY(NOW()) != 6, 2, 1) DAY ) +
```
Use explicit `AND` as `&` causes a false positive; I suspect `&` is evaluated as a bitwise operator and not logical `AND`